atd-vzd/README.md

```diff
@@ -1,18 +1,113 @@
+
+CRIS data accounts for the vast majority of records in the database: the [Vision Zero Editor (VZE)](../atd-vze/README.md) is designed primarily as a tool for editing and enriching data received from CRIS, and the [Vision Zero Viewer (VZV)](../atd-vzv/README.md) is powered entirely by enriched CRIS data.
+
+The CRIS data in our database consists of four record types:
+
+- `crashes` - each row is single crash event, with attributes such as the crash timestamp, crash location, and manner of collision
+- `units` - each row describes a unit, or vehicle, involved in a crash. Each unit relates to one crash record.
+- `people` - each row describes a person involved in a crash. Each person relates to one unit.
+- `charges` - each row descibes a legal charge filed by the responding law enforcement agency. Each charge relates to one person
+
+#### Design
+
+The core challenge that the Vision Zero database solves is to store CRIS data in a central repository where it can be reviewed and updated City of Austin staff. The database preserves the integrity of staff members' edits while simultaneously allowing crash record updates to flow into the database from CRIS.
+
+The editing environment is achieved by managing multiple copies of each of the `crashes`, `units`, and `people` tables, so that CRIS edits and Vision Zero staff edits remain isolated.
+
+For example, the `crashes` records are managed in three tables:
+
+- `crashes_cris`: stores crash records that are created and updated by TxdDOT CRIS through the [CRIS import ETL](../atd-etl/cris_import/README.md)
+- `crashes_edits`: stores crash record edits created by Visio Zero staff through the Vision Zero Editor web app
+- `crashes`: stores a unified version of each record which combines the values in `crashes_cris` plus any values in `crashes_edits`
+
+As pictured in the diagram below, the typical data flow for a crash record is as follows:
+
+1. A new record is inserted into the `crashes_cris` table through the [CRIS import ETL](../atd-etl/cris_import/README.md).
+2. On insert into `crashes_cris`, an "empty" copy of the record is inserted into the `crashes_edits` table. The record is inserted into the `crashes_edits` table with null values in every column except the `id`, which has a foreign key constaint referencing the `crashes_cris.id` column.
+3. At the same time, a complete copy of the record is inserted into the `crashes` table.
+4. A Vision Zero Editor user may update a crash records b updating rows in the the `crashes_edits` table. When an update is received, a trigger function coalesces each value in the `crashes_edits` table with the corresponding value in the `crashes_cris` table. The resulting record—which contains the original CRIS-provide values plus any edit values made through user edits—is applied as an update to corresponding record in the `crashes` table.
```

Comment:
there are just a few letters missing here and there in this line
b updating -> by updating rows
CRIS-provide -> CRIS-provided values